microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -12,12 +12,59 @@ ms.date: 02/23/2022
 This page lists the updates to Progressive Web Apps (PWAs) and Web Apps from the Microsoft Edge Web Apps team.  To try new features, read these announcements.  To stay up to date with the latest and greatest features, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download) (Beta, Dev, and Canary).
 
 
+<!-- ====================================================================== -->
+## What's New in Microsoft Edge 105
+
+Microsoft Edge 105 brings the following new features for Progressive Web Apps.
+
+
+### Window Controls Overlay
+
+The **Window Controls Overlay** API is now available by default with Microsoft Edge 105. This feature makes it possible for web developers to take control of the area that would normally be used by the title bar of an installed PWA window. The PWA code can use this area to render arbitrary web content such as a custom title, navigation buttons, or a menu bar.
+
+This feature makes PWAs installed on desktop operating systems more similar to their native counterparts.
```

Comment:
```suggestion
This feature makes PWAs that are installed on desktop operating systems more similar to their native counterparts.
```
I think style guide says to use non-condensed form to favor clarity over reduced word count.